### DISCUSSION TITLE
Inserting Images

### DISCUSSION THREAD

**Discussion by mingyuecall** [marp-team/marp#379] [Q&A] [answered]
Could I ask how to insert more than two images?
Thank you so much!

**Comment by yhatt** [comment, marked as answer]
Needless to say, [putting 3 Markdown images](https://www.markdownguide.org/basic-syntax/#images-1) will render 3 images.

```markdown
![](https://marp.app/favicon.png)
![](https://marp.app/favicon.png)
![](https://marp.app/favicon.png)
```
